client/lib/user-agent/index.js

```diff
@@ -0,0 +1,30 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { UserAgent } from 'express-useragent';
+import { get } from 'lodash';
+
+/*
+	# Clientside usage:
+
+	```
+	import userAgent from 'lib/user-agent';
+
+	const { isChromeOS, isIE } = userAgent;
+
+	if ( isChromeOS && isIE ) {
+		console.log( 'Hmm, this is unorthodox!' );
+	}
+	```
+
+	For a full list of values see: https://github.com/biggora/express-useragent/blob/master/lib/express-useragent.js#L191
+
+	Note: we also import this lib server-side in server/boot/index.js
+ */
+
+export default ( typeof window !== 'undefined' && !! get( window, 'navigator.userAgent', null )
+	? new UserAgent().parse( window.navigator.userAgent )
```

Comment:
nit: the package exports an instance of `UserAgent` as its default export. So this is possible, too:
```js
import UserAgent from 'express-useragent';
export default UserAgent.parse( navigator.userAgent );
```

Reply:
how expensive is the parse? this forces it to evaluate as part of the module load... we could do it on-demand if we exported a function instead.

Reply:
> how expensive is the parse? 

`window.performance` informs me that the parse takes between 4 and 5 milliseconds. This is in Chrome and Firefox on a fast Mac.